src/renderer/components/atoms/button/button.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/button-has-type */
+import type { ButtonProps } from './types';
+import { buildStylesFor } from './helper';
+
+const Button = ({
+  className,
+  label,
+  type = 'primary',
+  variant = 'big',
+  disabled = false,
+  children,
+}: ButtonProps) => {
+  const styles = buildStylesFor(type, variant, disabled);
+
+  if (disabled) {
+    return (
+      <button className={`${styles} ${className || ''}`} disabled>
```

Comment:
there is no need to have a separate `return` only if disabled is true. You can do this instead

```typescript
  return (
    <button className={`${styles} ${className || ''}`} disabled={disabled}>
      {label || children}
    </button>
  );
```

Reply:
Oh, right, sorry! Done!